boards/native/nrf_bsim/board_irq.h

```diff
@@ -18,6 +18,7 @@ void nrfbsim_WFE_model(void);
 void nrfbsim_SEV_model(void);
 
 #define IRQ_ZERO_LATENCY	BIT(1) /* Unused in this board*/
+#define IRQ_PRIO_LOWEST		UINT8_MAX
```

Comment:
```suggestion
#define IRQ_PRIO_LOWEST  UINT8_MAX
```

Reply:
you don't like the defines aligned? :)

Reply:
There is a tab, is it not? ... no. of spaces to align i can not make out in github, but definitely see a tab though.

Reply:
Tab only allowed at the beginning of lines in Zephyr, atleast that is how compliance used to be.